src/site/content/en/blog/generic-sensor/index.md

```diff
@@ -0,0 +1,478 @@
+  interface. The client code written for one sensor type can be
+  reused for another one with very few modifications!
+- You can configure the sensor, for example, set the sampling frequency suitable for your
+  application needs.
+- You can detect whether a sensor is available on the platform.
+- Sensor readings have high precision timestamps, enabling better
+  synchronization with other activities in your application.
+- Sensor data models and coordinate systems are clearly defined, allowing
+  browser vendors to implement interoperable solutions.
+- The Generic Sensor based interfaces are not bound to the DOM (neither
+  `navigator` nor `window` objects), and it opens up future opportunities of using
+  the same API within service workers or implementing Generic Sensor API in headless
+  JS runtimes, for instance, on embedded devices.
+- [Security and privacy](#privacy-and-security) aspects are the top priority
+  for the Generic Sensor API and provide much better security level compared to
+  older sensor APIs. There is integration with Permissions API.
+- Automatic [synchronization with screen coordinates](#synchronization-with-screen-coordinates) is
+  available for `Accelerometer`, `Gyroscope`, `LinearAccelerationSensor`, `AbsoluteOrientationSensor`,
+  `RelativeOrientationSensor` and `Magnetometer`.
+
+## Generic Sensor APIs in Chrome {: #generic-sensor-api-in-chrome }
+
+At the time of writing, Chrome supports several sensors that you can experiment with.
+
+**Motion sensors:**
+
+- `Accelerometer`
+- `Gyroscope`
+- `LinearAccelerationSensor`
+- `AbsoluteOrientationSensor`
+- `RelativeOrientationSensor`
+
+**Environmental sensors:**
+
+- `AmbientLightSensor`
+- `Magnetometer`
+
+The Generic Sensor-based motion sensor classes are enabled by default in Chrome
+starting from Chrome 67.
+
+You can enable environmental sensors for development purposes by turning on a
+feature flag. Go to
+[chrome://flags/#enable-generic-sensor-extra-classes](chrome://flags/#enable-generic-sensor-extra-classes)
+to enable environmental sensors. Restart Chrome and you should be good to go.
+
+More information on browser implementation status can be found on
+[chromestatus.com](https://www.chromestatus.com/features/5698781827825664?embed)
+
+## What are all these sensors? How can I use them? {: #what-are-sensors-how-to-use-them }
+
+Sensors is a quite specific area which might need a brief introduction. If you are familiar with
+sensors, you can jump right to the [hands-on coding section](#lets-code). Otherwise, let's look at
+each supported sensor in detail.
+
+### Accelerometer and linear acceleration sensor {: #acceleration-and-linear-accelerometer-sensor }
+
+<div class="w-figure w-figure--inline-right" style="max-width:50%">
+  <figure>
+    <img  src="accelerometer.gif"
+         alt="Accelerometer sensor measurements">
+    <figcaption class="w-figcaption"><b>Figure 1</b>: Accelerometer sensor measurements</figcaption>
```

Comment:
```suggestion
    <figcaption class="w-figcaption">Accelerometer sensor measurements</figcaption>
```

Reply:
I can't find any text that refers to a figure by its number, and web.dev doesn't number figures. Can you please remove them throughout.